app/src/main/java/org/imozerov/streetartview/ui/DetailOpener.java

```diff
@@ -0,0 +1,11 @@
+package org.imozerov.streetartview.ui;
+
+/**
+ * Created by sergei on 08.02.16.
+ */
+public interface DetailOpener {
+
+    void openDetailActivity();
```

Comment:
We have to pass ArtObject id to this method to be able to distinguish what object we need to show.